migrations/versions/07ebe99161d5_add_banner_image_url_to_sessio.py

```diff
@@ -0,0 +1,23 @@
+"""add banner_image_url field to session
+
+Revision ID: 07ebe99161d5
+Revises: d6b1904bea0e
+Create Date: 2018-11-21 19:06:35.140390
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '07ebe99161d5'
```

Comment:
I already have a PR with a migration ready to merge. This needs to be updated if I merge mine first. Or if this goes first then I'll update my PR.